stroyprombeton/settings/dev.py

```diff
@@ -14,17 +16,14 @@
 
 USE_CELERY = False
 
-
-"""
-    @todo #106 Включить в dev-среде django-debug-toolbar по-умолчанию
-     Но в CI его нужно отключать, т.к. возникают ошибки
-
-     def show_toolbar(request):
-         # Always display debug toolbar when running on development config
-         return True
-
-     DEBUG_TOOLBAR_CONFIG = {
-         'SHOW_TOOLBAR_CALLBACK': show_toolbar,
-     }
-
-"""
+def show_toolbar(request):
+    # Display debug toolbar when running on development config
+    # With exception for test environment
+    is_in_test_env = environ.get('TEST_ENV')
```

Comment:
1. `environ.get('TEST_ENV', False)` would be more idiomatic. It has the same functionality, but you leave explicit default value with the same type
2. we can drop if. Just do `return not environ.get('TEST_ENV', False)`